doc/source/cesm_configurations.rst

```diff
@@ -1,11 +1,11 @@
 .. _configurations:
 
 ===============================
-CESM Configurations (|version|)
+CESM2 Configurations (|version|)
```

Comment:
Actually, this is another place where the "CESM2" is particularly redundant. But again, I don't have strong feelings.